### PR TITLE
Modified clean & concise solution with One Loop

### PR DESCRIPTION
**Pull request by vigneshshiv**
Clean and concise solution with one loop. Please check if this is good to merge..

### CODE REVIEW

File: solutions/0000-0099/0003-longest-substring-without-repeating-characters-medium.md [hunk 1]
```diff
@@ -45,35 +45,32 @@ Notice that the answer must be a substring, "pwke" is a subsequence and not a su
...
 
-Create 2 pointer left and right place them at start of the string.
-Move right (right++) till distinct characters and store them in set.
-If repeated character occurs then move left (left++) until that repeated character is occured in left ans also remove all characters that occur before that character including character itself from set.
+Two pointer _i_ and _j_, initially at the start of the string.
+Move right (j++) till distinct characters and store them in set.
+If repeated character occurs then move left (i++) until that repeated character is occured in left ans also remove all characters that occur before that character including character itself from set.
+
+Time complexity: O(n), where n - # of characters in the string
+Space complexity: O(s), where s is the longest substring
```
Comment (wingkwong) [inline solutions/0000-0099/0003-longest-substring-without-repeating-characters-medium.md:55]: - Apply LaTex. 
- Use O instead of 0.
- Line break

**Review by wingkwong — CHANGES_REQUESTED** [review]
1. Run it locally to verify it first. 
image
2. Don't override others' solution. Add it below and point out how to optimize the above solution instead.

**Comment by vigneshshiv** [comment]
As suggested, I have modified the file and added another approach, Please take a look if it's good to go. 

The solutions are well tested and accepted in Leetcode, sample test cases found in my repo for the same problem. 
[Link for Testcase solution](https://github.com/vigneshshiv/leetcode/blob/main/code/java/sliding_window/LongestSubstringWithoutRepeatingChars.java)

File: solutions/0000-0099/0003-longest-substring-without-repeating-characters-medium.md (hunk 1 above)
Comment (vigneshshiv) [reply solutions/0000-0099/0003-longest-substring-without-repeating-characters-medium.md]: Modified the file and updated with the requested changes. Please verify once.

File: solutions/0000-0099/0003-longest-substring-without-repeating-characters-medium.md [hunk 2]
```diff
@@ -78,4 +116,44 @@ class Solution {
 }
 ```
 </TabItem>
+</Tabs>
+
+
+## Approach 2: Sliding Window with ASCII
```
Comment (wingkwong) [inline solutions/0000-0099/0003-longest-substring-without-repeating-characters-medium.md:122]: It should be Approach 3

**Review by wingkwong — CHANGES_REQUESTED** [review]
also add your name in Author in L3

File: solutions/0000-0099/0003-longest-substring-without-repeating-characters-medium.md (hunk 2 above)
Comment (vigneshshiv) [reply solutions/0000-0099/0003-longest-substring-without-repeating-characters-medium.md]: Updated and pushed it.

**Comment by vigneshshiv** [comment]
Updated with last suggestion, please take a look if this is Ok

**Review by wingkwong — APPROVED** [review]

**Pull request merged**